### DISCUSSION TITLE
Linux friendly and config?

### DISCUSSION THREAD

**Discussion by reevestrife** [keewek/vscode-compare-helper#7] [Q&A]
By default it seems the extension does not automatically create the config file for itself, when adding to visual studio code on ubuntu.  I also see it in the documentation there is only OS X and Windows example configurations.

So question:  

1. Is this extension working and tested in Linux?
2. Where should the config file be created for the extension to make it work?

**Comment by keewek** [comment]
1. Yes, I am using it on Linux too. Specifically, Xubuntu 22.04 LTS, Snap versions of VS Code and VSCodium.
2. No configuration files are created automatically. Extension can be configured via ```File --> Preferences --> Settings```. Much easier directly thru JSON file.
